code/modules/cargo/packsrogue/bandit/mage.dm

```diff
@@ -3,51 +3,281 @@
+/datum/supply_pack/rogue/bandit/Mage/paddedgambeson
+	name = "Padded Gambeson"
+	cost = 40
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/clothing/suit/roguetown/armor/gambeson/heavy)
+
+/datum/supply_pack/rogue/bandit/Mage/leatherstudded
+	name = "Studded Leather Armor"
+	cost = 50
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/clothing/suit/roguetown/armor/leather/studded)
+
+/datum/supply_pack/rogue/bandit/Mage/leathercoatheavy
+	name = "heavy leather coat"
+	cost = 60
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/clothing/suit/roguetown/armor/leathercoat/heavy)
+//spears
 
-//Weapons
 /datum/supply_pack/rogue/bandit/Mage/woodenstaff
 	name = "Wooden Staff"
-	cost = 20
+	cost = 5
 	contains = list(/obj/item/rogueweapon/woodstaff)
 
 /datum/supply_pack/rogue/bandit/Mage/spear
 	name = "Iron Spear"
-	cost = 50
+	cost = 25
 	contains = list(/obj/item/rogueweapon/spear)
 
+/datum/supply_pack/rogue/bandit/Mage/billhook
+	name = "Billhook"
+	cost = 40
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/rogueweapon/spear/billhook)
+
+/datum/supply_pack/rogue/bandit/Mage/bardiche
+	name = "Bardiche"
+	cost = 55
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/rogueweapon/halberd/bardiche)
+
+/datum/supply_pack/rogue/bandit/Mage/Lucerne
+	name = "Lucerne"
+	cost = 65
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/rogueweapon/eaglebeak/lucerne)
+
+/datum/supply_pack/rogue/bandit/Mage/halberd
+	name = "Halberd"
+	cost = 75
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/rogueweapon/halberd)
+
+/datum/supply_pack/rogue/bandit/Mage/ebeak
+	name = "Eagle's Beak"
+	cost = 75
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/rogueweapon/eaglebeak)
+//misc
+
+/datum/supply_pack/rogue/bandit/Mage/scrolls
+	name = "Scrolls"
+	cost = 3
+	contains = list(/obj/item/paper/scroll)
+
+/datum/supply_pack/rogue/bandit/Mage/infernalfeather
+	name = "Infernal Feather"
+	cost = 50
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/natural/feather/infernal)
+
+/datum/supply_pack/rogue/bandit/Mage/sendingstone
+	name = "Sending Stone"
+	cost = 50
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/sendingstonesummoner)
+
+/datum/supply_pack/rogue/bandit/Mage/scryorb
+	name = "Scrying Orb"
+	cost = 150
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/scrying)
+
+/datum/supply_pack/rogue/bandit/Mage/nomag
+	name = "Ring of Nullmagic"
+	cost = 200
+	time_lock = 1.5 HOURS
+	contains = list(/obj/item/clothing/ring/active/nomag)
+
+/datum/supply_pack/rogue/bandit/Mage/adeptspellbook
+	name = "Adept Spellbook"
+	cost = 150
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/book/granter/spellbook/adept)
+
+/datum/supply_pack/rogue/bandit/Mage/expertspellbook
+	name = "Expert Spellbook"
+	cost = 200
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/book/granter/spellbook/expert)
 
 /datum/supply_pack/rogue/bandit/Mage/mercury
 	name = "bottle of mercury"
+	cost = 25
+	contains = list(/obj/item/reagent_containers/glass/bottle/mercury)
+
```

Comment:
I'm also a little worried about enchantments being widely available like this. Getting enchantments takes considerable time on mages parts, never mind resources. None of the ones here are /overly/ egregious, but it's still something I'm... fairly skeptical over. It might be worth increasing it's prices.